website/content/docs/secrets/gcp.mdx

```diff
@@ -40,22 +40,77 @@ management tool.
 
 1.  Enable the Google Cloud secrets engine:
 
-    ```shell-session
+<Tabs>
+
+<Tab heading="CLI">
+
+   ```shell-session
```

Comment:
Remove indenting for the codeblock. For example:
````
<Tab heading="CLI">

```shell-session
$ vault secrets enable gcp
Success! Enabled the gcp secrets engine at: gcp/
```
````

The markdown parser is aggressive about interpreting indents as unordered lists. So it's interpreting the codeblock as an unordered list. Unfortunately, it also gets confused when parsing codeblocks (the formatting has to be specific to make it work). Since this isn't actually a list, removing the identing will solve the problem.